docs/pages/identity-governance/access-requests/automatic-reviews.mdx

```diff
@@ -90,36 +71,90 @@ does not support wildcard or regular expressions.
   by the IdP.
 </Admonition>
 
-## Step 3/4. Create automatic review rule
+## Step 3/5. Ensure resource labels
+
+List your SSH servers to identify available labels:
+
+```code
+$ tsh ls
+Node Name   Address    Labels
+----------- ---------- -------------------------------
+teleport-00 ⟵ Tunnel   env=demo
+```
+
+This example resource has the label `env: demo`. If your resource has not been
+labeled, reconfigure your resource with a label. Alternatively, you can add a
+label using a `server_info` resource.
```

Comment:
Should we collapse the `server_info` part by default? Ideally the reader doesn't have to learn a completely separate concept (dynamic server labels) in order to learn how to use automatic access reviews.